### ISSUE TITLE
Display gets 100% brightness on wakeup (Linux, KDE)

### ISSUE THREAD

**Issue by boredsquirrel** [Dasharo/dasharo-issues#772]
### Component

Dasharo firmware

### Device

NovaCustom NV4x 11th Gen

### Dasharo version

1.5.2

### Dasharo Tools Suite version

1.2.20

### Brief summary

When waking up from display off, the display is at 100% brightness

### How reproducible

Always.

### How to reproduce

In KDE I have set 1min to turn off the display. When I then move the cursor etc, the display turns on and has 100% brightness.

But it is still detected as 0% (for example) in KDE, setting it one step darker in the GUI settings makes it go to the correct level again. This will probably also happen when setting it one step higher.

### Expected behavior

The display should have the brightness that the OS sets

### Actual behavior

The brightness seems to not be read correctly or something, so that it is at 100% when waking up, until the OS sends another "change brightness" message

### Additional context

Likely not related to https://github.com/Dasharo/dasharo-issues/issues/297

### Solutions you've tried

Manually changing the brightness works, needs to be repeated always.

**Comment by macpijan** [comment]
@mkopec may be related to the linux driver as well?

**Comment by mkopec** [comment]
I don't think so, the brightness is handled using a generic ACPI interface and should just work without extra drivers. We need to reproduce this first, I think I haven't seen this when using GNOME but I may be wrong.

**Comment by boredsquirrel** [comment]
I can still reproduce this
- Fedora Kinoite 40
- KDE Plasma 6

the button combo can only go to 5%, while the slider can go to 1%. I have not checked if the wakeup brightness thing still persists

**Comment by philipanda** [comment]
> the button combo can only go to 5%, while the slider can go to 1%.

It's the same on my Lenovo Yoga with stock firmware on Cachy with KDE Plasma 6, so that's probably not a Dasharo issue.
